### DISCUSSION TITLE
Dragon's Breath, animation to spawn broken Ender Crystals when the dragon is revived, End Citys and End Boats

### DISCUSSION THREAD

**Discussion by GilberGames** [Roadhog360/Et-Futurum-Requiem#289] [Ideas]
Add the Dragon's Breath normal method (Right click when is in the floor) and the animation for spawning a new Dragon, the structures from the end will be great like the portal for the end citys.
P.D: Currently there is a natural way to find the shulkers?

**Comment by DarianLStephens** [comment]
There's an option to spawn shulkers naturally in the End, but that's currently it.

**Reply by GilberGames** [reply]
Thanks for the info!
